FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -131,10 +131,22 @@
 /**
  When @c YES appends the name of the device model and OS to the snapshot file name.
  The default value is @c NO.
+
+ @see includeOSVersionInFilename
  */
 @property (readwrite, nonatomic, assign, getter=isDeviceAgnostic) BOOL deviceAgnostic;
 
 /**
+ Used in conjunction with @c deviceAgnostic, when @c YES the OS version will be included in the snapshot file name.
+ The default value is @c YES.
+
+ @attention @c deviceAgnostic needs to be set to @c YES for this to take effect
+
+ @see deviceAgnostic
+ */
+@property (readwrite, nonatomic, assign) BOOL includeOSVersionInFilename;
```

Comment:
Can we change this from a `BOOL` to an enum where we could select what information to append? The default value will accommodate all of them. I want to avoid a specific property just for the name OS version.

```
typedef NS_OPTIONS(NSInteger, FBSnapshotTestDeviceAgnosticOption) {
  FBSnapshotTestDeviceAgnosticOptionModel         = 1 << 0,
  FBSnapshotTestDeviceAgnosticOptionSystemVersion = 1 << 1,
  FBSnapshotTestDeviceAgnosticOptionScreenSize    = 1 << 2,
};

@property (readwrite, nonatomic, assign) FBSnapshotTestDeviceAgnosticOption deviceAgnosticOptions; 
```